pkg/cmd/update/update.go

```diff
@@ -0,0 +1,94 @@
+package update
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"runtime"
+
+	"github.com/litmuschaos/litmusctl/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+var UpdateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "Changes the version of litmusctl",
+	Args:  cobra.ExactArgs(1),
+	Long:  ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		updateVersion := args[0]
+		var homeDir = "/usr/local/bin/"
```

Comment:
I think we need to specify some different path for windows. You can update this in windows switch case. You can create a custom directory for storing. Also it will be great if we can add this PATH to environment variables. We can use backslashes and the path will look maybe something like this` var homeDir = "C:\\litmusctl\\or\\anything\\else\\you can mention"`

Reply:
Hardcoding the homeDir is not the correct approach. You can use the package `"github.com/mitchellh/go-homedir"` for finding the home directory for the executing user. This package is already being used in litmusctl